mark-me-up-1.html

```diff
@@ -1,11 +1,42 @@
-This is basic HTML file. But right now, all it has is some text dumped into it.
-Your mission is to mark up this text so that it is more structured, readable and
-semantic. 
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>HTML markup into structured, readable and semantic text</title>
+    </head>
 
-HTML is the language of the web. It is used to create web pages. To quote the great Sir John Doe from https://johndoequotes.com:
-
-HTML is like water (H2O) - You can't see it, but that is the "life" of a web page.
-
-For more information, you can follow:
-1. The specification at https://html.spec.whatwg.org/
-2. MDN documentation at https://developer.mozilla.org/en-US/docs/Web/HTML
\ No newline at end of file
+    <body>
+        <h4>
+            This is basic HTML file. But right now, all it has is some text dumped
+            into it. Your mission is to mark up this text so that it is more
+            structured, readable and semantic.
+        </h4>
+        <p id="para1">
+            <strong><abbr>HTML</abbr></strong> (Hyper Text Markup Language) is the
```

Comment:
`title` attribute missing on `abbr`. it more a tooltip hint to visual users and full form to screen readers